src/app/item.service.ts

```diff
@@ -1,27 +1,45 @@
 import { Injectable } from '@angular/core';
 import { Observable, of} from 'rxjs';
+import { AngularFireModule } from '@angular/fire';
+import { AngularFirestore, AngularFirestoreCollection } from '@angular/fire/firestore';
+//import {AutoId} from '@firebase/firestore/dist/rn/firestore/src/util/misc';
+
+import {item} from './Item';
+import {itemSport} from './ItemSport';
+import {itemBook} from './ItemBook';
+import {itemClothing} from './ItemClothing';
+import {itemElectronic} from './ItemElectronic';
+import {itemFurniture} from './ItemFurniture';
 
-import { item } from './Item';
 import { Mock } from './MockBook';
 
 @Injectable({
   providedIn: 'root'
 })
 export class ItemService {
 
-  constructor() { }
+  private storedDatabase: AngularFirestoreCollection<item>
+  items$: Observable<item[]>
+  constructor(private firestore: AngularFirestore){
+    this.storedDatabase = this.firestore.collection('items');
+    this.items$ = this.storedDatabase.valueChanges();
+  }
 
   getItems(): Observable<item[]>{
   	// Currently returns Mock Item list 
```

Comment:
`getItems()` is unnecessary since `items$: Obersvable<item[]>` is assigned in the constructor. `items$` will provide (for the most part) an up to date `item[]` to all the components that use the `ItemService`.